ndcube/asdf/resources/schemas/ndcube-0.1.0.yaml

```
@@ -14,7 +14,11 @@ properties:
   data:
     description: "Must be compatible with ASDF serialization/deserialization and supported by NDCube."
   wcs:
-    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+    anyOf:
+      - tag: "tag:stsci.edu:gwcs/wcs-1.*"
+      - tag: "tag:sunpy.org:ndcube/resampledwcs-0.1.0"
+      - tag: "tag:sunpy.org:ndcube/reorderedwcs-0.1.0"
+      - tag: "tag:sunpy.org:ndcube/compoundwcs-0.1.0"
```

I don't think we need to be strict with the versions here at all.
```suggestion
      - tag: "tag:sunpy.org:ndcube/resampledwcs-*"
      - tag: "tag:sunpy.org:ndcube/reorderedwcs-*"
      - tag: "tag:sunpy.org:ndcube/compoundwcs-*"
```